src/mmwtex.c

```diff
@@ -1222,7 +1222,6 @@ vstring getCommentModeSection(vstring *srcptr, char *mode)
     }
     ptr++;
   } // while (1)
-  assert(0);
```

Comment:
Am I correct ? This follows a `while (1)`  loop with no `break;` statement in the outermost loop (the `break;` is in a `switch` statement within the `while(1)` loop). Therefore, this is unreachable code.

Reply:
The `break;` after `return modeSection;` looks very strange in that code, that is also clearly unreachable code.

Reply:
Done in 6523fc7c1a292fdbba145b11d65a5d2f876e518f
I didn't go further and didn't change the `switch` statement since I don't know the code enough (`mode` seems to take "magic" values, for instance).